docs/AddUnitTests.md

```diff
@@ -48,14 +48,49 @@ will just tell you that `TryGetValue` failed, without telling you the key it was
 
 The output will now tell you that `IsTrue` failed, but also why. This is especially handy in cases where your `Assert` statement appears within a loop
 
-### Working with Fakes
-Due to the known issue (overwriting fakes assemblies from different unit tests if they have same name), all `Fakes` assemblies are generated in a single project (`AccessibilityInsights.Fakes.Prebuild`). 
+### Working with Microsoft Fakes
 
-If you need to add any new `Fake` assembly, please follow the below steps:
+#### Adding a test assembly that requires Fakes
+
+Due to the known issue (overwriting fakes assemblies from different unit tests when they have the same name), all `Fakes` assemblies are generated in a single project (`AccessibilityInsights.Fakes.Prebuild`). 
+
+If you need to add a new `Fake` assembly, please follow the below steps:
 1. Add a new fakes file into `AccessibilityInsights.Fakes.Prebuild`. 
 2. Build `AccessibilityInsights.Fakes.Prebuild`.
 3. Add a project dependency (not project reference) of `AccessibilityInsights.Fakes.Prebuild` into the unit test which needs a new fakes assembly.
 4. Add an assembly reference of *.fakes.dll (or exe) in `AccessibilityInsights.Fakes.Prebuild/FakesAssemblies` folder to the unit test.
 5. Add `Microsoft.QualityTools.Testing.Fakes.dll` into the assembly reference of the unit test.
+6. Any references to Fakes assemblies which are added to any test projects should be conditional on the environment variable `FAKES_SUPPORTED = 1`. See the next section for details.
+
+#### Important: Not all editions of Visual Studio support Fakes
+
+Some of the unit tests in Accessibility Insights For Windows use Microsoft Fakes to replace system calls which are otherwise difficult to mock using standard mocking frameworks. The problem is that certain versions of Visual Studio, like Enterprise, support Fakes. And some, like Community, do not. Because Accessibility Insights is open source, we cannot predict which versions of Visual Studio contributers will have access to. Therefore, we have disabled the use of Fakes by default.
```

Comment:
slight typo, contributers should be contributors 
```suggestion
Some of the unit tests in Accessibility Insights For Windows use Microsoft Fakes to replace system calls which are otherwise difficult to mock using standard mocking frameworks. The problem is that certain versions of Visual Studio, like Enterprise, support Fakes. And some, like Community, do not. Because Accessibility Insights is open source, we cannot predict which versions of Visual Studio contributors will have access to. Therefore, we have disabled the use of Fakes by default.
```